file.go

```diff
@@ -758,6 +763,10 @@ func (f *FilePages) init(c *FileColumnChunk, reader io.ReaderAt) {
 	f.section = *io.NewSectionReader(reader, f.baseOffset, c.chunk.MetaData.TotalCompressedSize)
 	f.rbuf, f.rbufpool = getBufioReader(&f.section, f.bufferSize)
 	f.decoder.Reset(f.protocol.NewReader(f.rbuf))
+	f.index = 0
+	f.lastPage = nil
```

Comment:
should this free last page if it's non-nil? FilePages is not actually re-used but this "init" makes it seem like it wants to be. possibly someone will do this in the future.